Bootstrap/Bootstrap-OpenSymbols.svg

```diff
@@ -0,0 +1,4038 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="svg1479"
+   sodipodi:docname="Bootstrap-icons.svg"
+   inkscape:version="0.92.4 (unknown)">
+  <title
+     id="title102576">Bootstrap Icons</title>
+  <metadata
+     id="metadata1485">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title>Bootstrap Icons</dc:title>
+        <dc:date>04 Feb 2020</dc:date>
+        <dc:creator>
+          <cc:Agent>
+            <dc:title>meet</dc:title>
+          </cc:Agent>
+        </dc:creator>
+        <dc:rights>
+          <cc:Agent>
+            <dc:title>Custom</dc:title>
+          </cc:Agent>
+        </dc:rights>
+        <dc:publisher>
+          <cc:Agent>
+            <dc:title>Twitter Bootstrap</dc:title>
+          </cc:Agent>
+        </dc:publisher>
+        <dc:source>https://github.com/twbs/icons/releases/tag/v1.0.0-alpha2</dc:source>
+        <dc:subject>
+          <rdf:Bag>
+            <rdf:li>Inkscape</rdf:li>
+            <rdf:li>symbol</rdf:li>
+          </rdf:Bag>
+        </dc:subject>
+        <dc:description>Inkscape symbol pack based on Bootstrap icons</dc:description>
+        <dc:contributor>
+          <cc:Agent>
+            <dc:title>meet</dc:title>
+          </cc:Agent>
+        </dc:contributor>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+
+
+  <symbol
+     id="alarm-fill"
+     viewBox="0 0 20 20">
+  
+  </symbol>
+
+    <path
+       fill-rule="evenodd"
+       d="M10 17a6 6 0 100-12 6 6 0 000 12zm0 1a7 7 0 100-14 7 7 0 000 14z"
+       clip-rule="evenodd"
+       id="path5" />
+    <path
+       fill-rule="evenodd"
+       d="M10 6.5a.5.5 0 01.5.5v4a.5.5 0 01-.053.224l-1.5 3a.5.5 0 11-.894-.448L9.5 10.882V7a.5.5 0 01.5-.5z"
+       clip-rule="evenodd"
+       id="path7" />
+    <path
+       d="M2.86 7.387A2.5 2.5 0 116.387 3.86 8.035 8.035 0 002.86 7.387zM13.613 3.86a2.5 2.5 0 113.527 3.527 8.035 8.035 0 00-3.527-3.527z"
+       id="path9" />
+    <path
+       fill-rule="evenodd"
+       d="M13.646 16.146a.5.5 0 01.708 0l1 1a.5.5 0 01-.708.708l-1-1a.5.5 0 010-.708zm-7.292 0a.5.5 0 00-.708 0l-1 1a.5.5 0 00.708.708l1-1a.5.5 0 000-.708zM7.5 2.5A.5.5 0 018 2h4a.5.5 0 010 1H8a.5.5 0 01-.5-.5z"
+       clip-rule="evenodd"
+       id="path11" />
+    <path
+       d="M9 3h2v2H9V3z"
+       id="path13" />
+  </symbol>
```

Comment:
Did you test it? Seems like the set is not well formed.  This symbol is wrongly closed in line 63